src/scripts/helpers/handlebars/trim.coffee

```diff
@@ -5,7 +5,12 @@ define (require) ->
     temp = document.createElement("div")
     temp.innerHTML = str
     str = temp.textContent
-    str.replace(/\s/g, '-').replace(/[^a-zA-Z0-9 -]/g, '').substring(0, 30)
+    # replace symbols with whitespace
+    str = str.replace(/[\u2000-\u206F\u2E00-\u2E7F\\'!"#$%&()*+,\-.\/:;<=>?@\[\]^_`{|}~\s+]/g, ' ')
```

Comment:
The `\s+` in this line is probably unnecessary (since the last line condenses all spaces into dashes) but it's a minor thing so I'll approve anyway.